src/sa_web/static/js/views/app-view.js

```diff
@@ -41,9 +41,11 @@ var Shareabouts = Shareabouts || {};
 
       // Boodstrapped data from the page
       this.activities = this.options.activities;
-      this.places = this.collection;
+      this.places = this.collection.places;
       this.landmarkCollections = this.options.landmarkCollections;
 
+      self.collection = this.options.collection;
+
```

Comment:
I am a little confused here, because I thought  we were making `self.collection` a hash that contains both places and landmarks. If so, then wouldn't we get rid of `this.places` and `this.landmarkCollections`, and use `this.collection` instead?